cypress/integration/a11y.spec.js

```diff
@@ -0,0 +1,200 @@
+describe('Check accessiblity', () => {
+    const TOTAL_ELIBIGILITY_QUESTIONS = 3;
+
```

Comment:
you can destructure up here all of the methods that you're using frequently like 
`const { findByLabelText, findByRole, setCookie } = cy` to improve the readability